critiquebrainz/frontend/views/bb_author.py

```diff
@@ -30,6 +31,28 @@ def entity(id):
             return redirect(url_for('bb_author.entity', id=redirected_bbid))
         raise NotFound(gettext("Sorry, we couldn't find an author with that BookBrainz ID."))
 
+    artist_info = {}
+    if author['identifiers']:
+        mb_artist_mbid = []
+        for identifier in author['identifiers']:
+            if identifier['name'] == 'MusicBrainz Artist ID':
+                mb_artist_mbid.append(identifier['value'])
+
+        if mb_artist_mbid:
+            artist_info = fetch_multiple_artists(mb_artist_mbid)
+            for artist in artist_info:
+                reviews, count = db_review.list_reviews(
```

Comment:
This bit of code is a candidate for an optimisation, although in the case that we currently have for these pages this isn't necessary.

A common error in designing database queries is to perform 1+n queries - i.e. one query to get a list of items, and then for each of those n items, another query for each of them. This could result in thousands of queries being made, when it could have been simplified to perform just 1 query (for example to get the list of items and their related data) by changing the query or including a join to another table.

In this case, we could have multiple mbids, and then for each of them we call the `list_reviews` method (this actually makes at least 2 sql queries, one to get the count of all reviews, and another to get the actual reviews). This means that if we had 10 artist mbids, we'd have to make 20 sql queries, all of them more or less the same except for the entity id.
However, in the specific case of BB-MB relationships, we don't expect to see more than 1 or 2 related MBIDs, so we're not going to make too many database queries.

In the future, we may have more requirements to get information/reviews about multiple entities at once. I'll open a ticket to add this feature so that we know that we might need it.

For this PR, no requirement to do anything else, I was just letting you know about the problem.